layers/+filetree/treemacs/README.org

```diff
@@ -79,7 +80,20 @@ layer variable =treemacs-use-collapsed-directories= to a positive number.
     (treemacs :variables treemacs-use-collapsed-directories 3)))
 #+END_SRC
 
-Default value is 3 (or 0 when python is not installed).
+Default is 3 (or 0 when python is not installed).
+
+** Locking width
+To have the width of the treemacs window locked by default, meaning not manually
+resizable.
+Note that it will still be resizable using =treemacs-toggle-fixed-width= or
```

Comment:
I think you should place an empty line here,

Reply:
You mean between the two sentences ?